src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
```

Comment:
Instead of using this constructor, will be better to remove this constructor, make all variables `private final` and also remove all setters of variables. After that all fields will be created with invoking of constructor with parameters